eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/service/UserService.java

```diff
@@ -33,12 +33,12 @@
 
   public UserResponseDTO getUser(Long id) {
     return new UserResponseDTO(userRepository.findById(id)
-        .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND)));
+        .orElseThrow(NotFoundException::new));
   }
 
   public UserResponseDTO createUser(UserRequestDTO userRequestDTO) {
     if (userRepository.findFirstByEmail(userRequestDTO.getEmail()).isPresent()) {
-      throw new ResponseStatusException(HttpStatus.BAD_REQUEST);
+      throw new BadRequestException();
```

Comment:
Itt még mindig itt van ez a BadRequest dolog, ami megint csak controller-related. Dobjunk itt is not found exceptiont.
Btw, miért nem használjuk itt ki az Optionalokat? Ne futtassunk le plusz lekérdezést feleslegesen.

Reply:
Itt akkor dob bad request-et, ha létezik már a megadott email cím. Unique kell hogy legyen.